tests/fiber_abstraction/floating_point_fiber_switch.cpp

```diff
@@ -152,16 +152,24 @@ void FloatingPointSixthLevelFiberStart(void *arg) {
 	FAIL();
 }
 
+constexpr static std::size_t HALF_MEBIBYTE = 524288;
```

Comment:
Hmm, perhaps `k500MebiByte` or `k500Mebibyte`

Reply:
Will be applied in a commit I'll make in a bit.